app/src/main/res/layout/row_chat_history.xml

```diff
@@ -5,6 +5,9 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_gravity="top"
```

Comment:
we have 2 `android:id="@+id/chatCardView"` causing the crash

Reply:
I saw this as well. Getting rid of it now